Documentation/project-docs/changing-corelib.md

```diff
@@ -1,22 +1,28 @@
 
-**Staging the changes**
+## How to stage your change
 
-Make the changes in both CoreCLR and CoreFX
-- System.Private.CoreLib implementation changes should should be made in CoreCLR repo
+### (1) Make the changes in both CoreCLR and CoreFX
+- `System.Private.CoreLib` implementation changes should should be made in CoreCLR repo
 - Test and public API contract changes should be made in CoreFX repo
 - [Build and test](https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/developer-guide.md#testing-with-private-coreclr-bits) both changes together
 
-Submit PR to both CoreCLR and CoreFX
-- Link the two PRs together via comment in PR description
+### (2) Submit PR to both CoreCLR and CoreFX
+- Link the two PRs together via comment in PR description, and link both to the issue itself.
 - Both PRs will reviewed together by the project maintainers
-- CoreFX tests run on CoreCLR PRs are using old published build. You may need to disable the outdated tests in https://github.com/dotnet/coreclr/blob/master/tests/CoreFX/CoreFX.issues.json to make your PR green.
-- CoreCLR PR will be merged first
-- If your change is under [System.Private.CoreLib Shared Sources](https://github.com/dotnet/coreclr/tree/master/src/System.Private.CoreLib/shared), it will get mirrored to other repos that are reusing the CoreLib sources.
-- Once updated CoreCLR is consumed by CoreFX repo, CoreFX PR will be merged second.
+- CoreCLR runs CoreFX tests but they are behind CoreFX. You may need to disable the outdated tests in https://github.com/dotnet/coreclr/blob/master/tests/CoreFX/CoreFX.issues.json to make your PR green.
+
+### (3) What happens next
+- We will merge the CoreCLR PR first
+- Note: if your change is under [System.Private.CoreLib Shared Sources](https://github.com/dotnet/coreclr/tree/master/src/System.Private.CoreLib/shared), it will get mirrored to other repos that are reusing the CoreLib sources. This is a one-way mirror of sources for code reuse purposes: it does not bring your new API to CoreFX so it is not relevant to this staging process.
+- The CoreCLR changes will be consumed by CoreFX via an automatically created PR that updates a hash in the CoreFX repo. These PR's [look like this](https://github.com/dotnet/corefx/pulls?utf8=%E2%9C%93&q=is%3Apr+sort%3Aupdated-desc+coreclr++base%3Amaster+author%3Adotnet-maestro-bot+)
+- Depending on the nature of the change, we may cherry-pick your CoreFX PR into this automatically created PR; or, we may merge your PR after we merge the automatically created PR.
+- Consider creating a new PR against CoreCLR to re-enable any tests you had to disable in your original CoreCLR PR. This can be merged when CoreCLR in turn consumes CoreFX with your chanbges.
```

Comment:
> Consider creating a new PR against CoreCLR to re-enable any tests you had to disable in your original CoreCLR PR. This can be merged when CoreCLR in turn consumes CoreFX with your chanbges.

This does not work today. We are running old fixed CoreFX drop today and thus you cannot re-enable the tests because the test updates are not showing up in the drop used by CoreCLR. (Progress on fixing this is blocked by Azure AD migration.)